tests/e2e/bento_server_general_features/train.py

```diff
@@ -0,0 +1,11 @@
+from pickle_model import PickleModel
+
+import bentoml.sklearn
```

Comment:
no need to do this with the recent changes

Reply:
actually i didn't import bentoml so it's needed.

Reply:
oops my b